### DISCUSSION TITLE
Is it possible to install on shared hosting

### DISCUSSION THREAD

**Discussion by johnroberts2k** [snipe/snipe-it#10391] [Q&A] [answered]
I have a shared hosting platform with access to mysql. Is it possible to install snipe-it on a shared hosting platform? If so can anyone kindly direct me on how I can do this? I've unzipped the source file on the hosting account but don't know what to do next

**Comment by snipe** [comment, marked as answer]
Shared hosting is addressed in the requirements docs: https://snipe-it.readme.io/docs/requirements
